docs/acme-protocol-updates.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: ACME Protocol Updates
+permalink: /docs/acme-protocol-updates/
+top_graphic: 1
+---
+
+[<- Back to Documentation List](/docs/)
+
+The ACME protocol is the cornerstone of how Let's Encrypt works. As the protocol specification evolves over time Let's Encrypt will implement updated versions of ACME. When doing so, security will be our primary concern, followed closely by backwards compatibility.
```

Comment:
Link to spec? Also somewhere in this document we should mention that we implement draft 01 of the spec, with some tweaks. Once we land the changes thus far, I can make some further edits to this doc explaining those tweaks.